h5pyd/_apps/utillib.py

```diff
@@ -591,68 +735,37 @@ def create_chunktable(dset, dset_dims, ctx):
         chunks["size"] = dset_size
     else:
         # construct map of chunks if count is less than 10
+        # get the chunk locations
```

Comment:
Is there a reason to remove the conditional use of `chunk_iter` with `create_chunktable_callback` here?

Reply:
Since this case is only a few chunks, it's simpler to just do get_chunk_info.